library/spdm_transport_pcidoe_lib/libspdm_doe_pcidoe.c

```diff
@@ -128,6 +128,61 @@ libspdm_return_t libspdm_pci_doe_encode_message(const uint32_t *session_id,
     return LIBSPDM_STATUS_SUCCESS;
 }
 
+/**
```

Comment:
Delete the doxygen comments from non-static functions in .c files that already have the doxygen comments in the function declaration.